Tests/GoodNetworkingTests/DateDecodingTest.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  DateDecodingTest.swift
+//  
+//
+//  Created by Matúš Klasovitý on 14/03/2023.
+//
+
+import Foundation
+import GoodNetworking
+import XCTest
+import Mockable
+
+final class DateDecodingTest: XCTestCase {
+        
+    struct IsoDateWrapper: Decodable, WithCustomDecoder {
+        
+        static var decoder: JSONDecoder {
```

Comment:
Trocha som sa nad tym zamyslal a vychadza mi ze mozno by bolo krajsie zaobalit tu strategiu.
Vies to aplikovat ako extension na WithCustomDecoder a zasa budeme mat ako predtym cez override. To mi prislo krajsie ako toto riesenie